tests/python/PyImportSortBearTest.py

```diff
@@ -21,3 +21,12 @@
      'import abc\nimport xyz\n'),
     settings={'known_standard_library_imports': 'xyz',
               'known_first_party_imports': 'abc'})
+
+PyImportSortBearIgnoredConfigsTest = verify_local_bear(
+    PyImportSortBear,
+    ('import sys\n\nimport datetime\n',
+     'from sys import *\n\nimport datetime\n'),
+    ('import sys\nimport datetime\n',
+     'import datetime\nimport sys\n'),
+    settings={'known_standard_library_imports': 'sys',
```

Comment:
Remove the `known_standard_library_imports': 'sys'`, `sys` is already a stdlib module.